tests/test_ioc_system.py

```diff
@@ -104,14 +104,14 @@ async def test_create_softioc_system(
         test_prefix,
     ) = mocked_panda_standard_responses
 
-    assert await caget(test_prefix + ":PCAP:TRIG_EDGE") == 1  # == Falling
-    assert await caget(test_prefix + ":PCAP:GATE") == "CLOCK1.OUT"
-    assert await caget(test_prefix + ":PCAP:GATE:DELAY") == 1
-
     for field_name, expected_array in table_unpacked_data.items():
         actual_array = await caget(test_prefix + ":SEQ:TABLE:" + field_name)
         assert numpy.array_equal(actual_array, expected_array)
 
+    assert await caget(test_prefix + ":PCAP:TRIG_EDGE") == 1  # == Falling
+    assert await caget(test_prefix + ":PCAP:GATE") == "CLOCK1.OUT"
+    assert await caget(test_prefix + ":PCAP:GATE:DELAY") == 1
+
     pcap1_label = await caget(test_prefix + ":PCAP:LABEL")
```

Comment:
I think that timing issues should not be affecting this test in the first place. This test never wants any data to change, all of its checks are just on the default data returned from the initial *CHANGES request. Hence, I think that the correct fix to this is to use different mocked data which just doesn't change at all. Then there won't be any timing issues.

Reply:
I'm also considering whether it'd be better to not do all these `caget` commands serially, and instead create them all as tasks and do an `asyncio.gather()`. But that's probably overly complicated.

Reply:
I've made a new mocked panda which doesn't update past its initial SEQ and PCAP values, and takes no put requests.

Reply:
> I'm also considering whether it'd be better to not do all these `caget` commands serially, and instead create them all as tasks and do an `asyncio.gather()`. But that's probably overly complicated.

Now that SEQ pvs are guaranteed to stay the same until the test ends I'd say sequentially cagetting is fairly readable and not a performance bottleneck. Happy to change it though, up to you.

Reply:
I'm happy to leave it as-is.